<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SAOMDListImageClass</class>
 <widget class="QMainWindow" name="SAOMDListImageClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1183</width>
    <height>880</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SAO:MD List Image </string>
  </property>
  <property name="windowIcon">
   <iconset theme=":/SAOMDListImage/icon">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <property name="dockOptions">
   <set>QMainWindow::AllowNestedDocks|QMainWindow::AllowTabbedDocks|QMainWindow::AnimatedDocks</set>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="CScaleControlView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1183</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_Load"/>
    <addaction name="actionClear"/>
    <addaction name="action_Save"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionDemo_Video"/>
    <addaction name="actionHow_to_use"/>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <addaction name="actionFit_Image"/>
     <addaction name="actionFit_Width"/>
     <addaction name="actionFit_Height"/>
     <addaction name="separator"/>
     <addaction name="actionZoomFull"/>
     <addaction name="actionZoomHalf"/>
     <addaction name="actionZoomCustom"/>
     <addaction name="separator"/>
     <addaction name="actionZoom_In"/>
     <addaction name="actionZoom_Out"/>
    </widget>
    <addaction name="menuZoom"/>
    <addaction name="separator"/>
    <addaction name="actionScene"/>
    <addaction name="actionUpdate_Checker"/>
    <addaction name="actionAuto_Capture"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockScene">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>524287</width>
     <height>335</height>
    </size>
   </property>
   <property name="windowTitle">
    <string>Scene</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="title">
        <string>View</string>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Zoom</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QSlider" name="hsZoom">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>200</number>
          </property>
          <property name="sliderPosition">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>10</number>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Layout</string>
       </property>
       <layout class="QGridLayout" name="gridLayout">
        <item row="2" column="0">
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Number</string>
          </property>
         </widget>
        </item>
        <item row="3" column="1">
         <widget class="QSlider" name="hsColumnNum">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="value">
           <number>16</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="tickPosition">
           <enum>QSlider::TicksAbove</enum>
          </property>
          <property name="tickInterval">
           <number>5</number>
          </property>
         </widget>
        </item>
        <item row="4" column="1">
         <widget class="QLineEdit" name="leArraySize">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="1">
         <widget class="QSlider" name="hsItemBorder">
          <property name="maximum">
           <number>50</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="leNumber">
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="5" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Border Size</string>
          </property>
         </widget>
        </item>
        <item row="3" column="0">
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Column number</string>
          </property>
         </widget>
        </item>
        <item row="6" column="0">
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Image Size</string>
          </property>
         </widget>
        </item>
        <item row="6" column="1">
         <widget class="QLineEdit" name="leImageSize"/>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Item Control</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QPushButton" name="pbDeleteSelected">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>30</height>
           </size>
          </property>
          <property name="text">
           <string>Delete Selected</string>
          </property>
          <property name="shortcut">
           <string>Ctrl+S</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockUpdateer">
   <property name="windowTitle">
    <string>Update Checker</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2"/>
  </widget>
  <widget class="QDockWidget" name="dockAutoMode">
   <property name="windowTitle">
    <string>Auto Capture</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3"/>
  </widget>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionScene">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Scene</string>
   </property>
  </action>
  <action name="actionUpdate_Checker">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Update Checker</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionItem_Control">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Item Control</string>
   </property>
  </action>
  <action name="actionDemo_Video">
   <property name="text">
    <string>Demo Video</string>
   </property>
  </action>
  <action name="actionHow_to_use">
   <property name="text">
    <string>How to use</string>
   </property>
  </action>
  <action name="actionFit_Image">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Image</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionFit_Width">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Width</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionFit_Height">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Fit Height</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="actionZoomFull">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>100%</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+0</string>
   </property>
  </action>
  <action name="actionZoomHalf">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>50%</string>
   </property>
  </action>
  <action name="actionZoomCustom">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Customize</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="shortcut">
    <string>Ctrl++</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+-</string>
   </property>
  </action>
  <action name="actionAuto_Capture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto Capture</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CScaleControlView</class>
   <extends>QGraphicsView</extends>
   <header>CScaleControlView.h</header>
   <slots>
    <signal>signalScaleChanged(float)</signal>
    <signal>signalModeChanged(CScaleControlView::EScaleMode)</signal>
    <slot>slotZoomFitImage()</slot>
    <slot>slotZoomFitWidth()</slot>
    <slot>slotZoomFitHeight()</slot>
    <slot>slotZoomScaleFull()</slot>
    <slot>slotZoomScaleHalf()</slot>
    <slot>slotZoomIn()</slot>
    <slot>slotZoomOut()</slot>
    <slot>setScale(float)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="SAOMDListImage.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_Load</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotLoad()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_Save</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotSave()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsColumnNum</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotColumnNumChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1163</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>881</x>
     <y>-14</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsItemBorder</sender>
   <signal>valueChanged(int)</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>updateLayout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1058</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>647</x>
     <y>-13</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAbout</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotAbout()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionScene</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockScene</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>133</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockScene</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionScene</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1048</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionUpdate_Checker</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockUpdateer</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>548</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockUpdateer</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionUpdate_Checker</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1048</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionClear</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotClear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbDeleteSelected</sender>
   <signal>clicked()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotDeleteSelected()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1048</x>
     <y>838</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionDemo_Video</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotDemoVideo()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionHow_to_use</sender>
   <signal>triggered()</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotHowToUse()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomFull</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomScaleFull()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFit_Width</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomFitWidth()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFit_Height</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomFitHeight()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFit_Image</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomFitImage()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoomHalf</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomScaleHalf()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_In</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomIn()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionZoom_Out</sender>
   <signal>triggered()</signal>
   <receiver>graphicsView</receiver>
   <slot>slotZoomOut()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>454</x>
     <y>446</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hsZoom</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotZoom(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1068</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphicsView</sender>
   <signal>signalModeChanged(CScaleControlView::EScaleMode)</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotModeChanged(CScaleControlView::EScaleMode)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>graphicsView</sender>
   <signal>signalScaleChanged(float)</signal>
   <receiver>SAOMDListImageClass</receiver>
   <slot>slotSclaeChanged(float)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>454</x>
     <y>446</y>
    </hint>
    <hint type="destinationlabel">
     <x>591</x>
     <y>439</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dockAutoMode</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionAuto_Capture</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1048</x>
     <y>840</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionAuto_Capture</sender>
   <signal>triggered(bool)</signal>
   <receiver>dockAutoMode</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>1048</x>
     <y>840</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slotLoad()</slot>
  <slot>slotSave()</slot>
  <slot>updateLayout()</slot>
  <slot>slotAbout()</slot>
  <slot>slotClear()</slot>
  <slot>slotDeleteSelected()</slot>
  <slot>slotDemoVideo()</slot>
  <slot>slotHowToUse()</slot>
  <slot>slotZoom(int)</slot>
  <slot>slotModeChanged(CScaleControlView::EScaleMode)</slot>
  <slot>slotSclaeChanged(float)</slot>
  <slot>slotColumnNumChanged(int)</slot>
 </slots>
</ui>
